4_Bayes_Rule/Bayes_Rule_All.ipynb

```diff
@@ -0,0 +1,4120 @@
+{
```

Comment:
<p>Maybe put this into a nice DataFrame/Series that links the conversion rates to probs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>oh good idea, way easier to read</p>